### DISCUSSION TITLE
The Map Control how to fly and zoom to one Point(lat,lon)?

### DISCUSSION THREAD

**Discussion by dingyichina** [flet-dev/flet#3705] [Q&A]
### Question

I am using the Map and Geolocator ，and I want to fly and zoom to my Location ，but I can't find the function to do this. How can I do ?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Sub to https://github.com/flet-dev/flet/issues/3688
